Arduino/victoria_teensy_ros_node/victoria_teensy_ros_node.ino

```diff
@@ -101,7 +113,7 @@ RosParamHelper ros_param_helper(ros_nh);
 // to stop robot activity
 double cmd_vel_timeout_threshold;
 ros::Time last_cmd_vel_time;
-bool cmd_vel_timeout_stop = false; // Set to true if time out threshold has been exceeded
+bool cmd_vel_timeout_exceeded = false; // Set to true if time out threshold has been exceeded
```

Comment:
This is an assignment to false, not an initialization. I know that for basic types there is no savings, but there is also no cost and it helps keep things consistent.